common/views/components/Image/Image.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import {convertImageUri} from '../../../utils/convert-image-uri';
+import {imageSizes} from '../../../utils/image-sizes';
+
+type Props = {|
+  width: string,
+  height?: string,
+  contentUrl: string,
+  clipPathClass?: string,
+  alt?: string,
+  caption?: string,
+  lazyload: ?boolean,
+  sizesQueries?: string,
+  copyright?: string,
+  defaultSize?: number
+|}
+
+const Image = ({width, height, contentUrl, clipPathClass, alt, caption, lazyload, sizesQueries = '100vw', copyright, defaultSize = 30}: Props) => (
+  <div className="work-media__image-container">
+    <noscript>
+      <img width={width}
+        height={height}
```

Comment:
As height is potentially null, do we still want to set it?
Not sure if it make a difference in some browser somewhere?

Reply:
Apparently giving a falsey value to an attribute will prevent it from being output in JSX (https://stackoverflow.com/a/31164090/590593), so height simply won't appear if there isn't a height value.